Cron/ShareOfCheckout.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Alma\MonthlyPayments\Cron;
+
+use Alma\API\RequestError;
+use Alma\MonthlyPayments\Helpers\DateHelper;
+use Alma\MonthlyPayments\Helpers\Logger;
+use Alma\MonthlyPayments\Helpers\ShareOfCheckoutHelper;
+
+
+class ShareOfCheckout
+{
+    /**
+     * @var Logger
+     */
+    private $logger;
+    /**
+     * @var ShareOfCheckoutHelper
+     */
+    private $shareOfCheckoutHelper;
+    /**
+     * @var DateHelper
+     */
+    private $dateHelper;
+
+    /**
+     * @param Logger $logger
+     * @param ShareOfCheckoutHelper $shareOfCheckoutHelper
+     * @param DateHelper $dateHelper
+     */
+    public function __construct(
+        Logger $logger,
+        ShareOfCheckoutHelper $shareOfCheckoutHelper,
+        DateHelper $dateHelper
+    )
+    {
+        $this->logger = $logger;
+        $this->shareOfCheckoutHelper = $shareOfCheckoutHelper;
+        $this->dateHelper = $dateHelper;
+    }
+
+    /**
+     * @return void
+     */
+    public function shareDays():void
+    {
+        ini_set('max_execution_time', 30);
```

Comment:
IIRC the default for this function is already 30s ([it is](https://www.php.net/manual/en/function.set-time-limit.php)). Did you wanted to override client config ?